pkg/graph/crd/crd.go

```diff
@@ -62,7 +63,7 @@ func newCRD(group, apiVersion, kind string, schema *extv1.JSONSchemaProps, addit
 					Subresources: &extv1.CustomResourceSubresources{
 						Status: &extv1.CustomResourceSubresourceStatus{},
 					},
-					AdditionalPrinterColumns: newCRDAdditionalPrinterColumns(additionalPrinterColumns),
+					AdditionalPrinterColumns: newCRDAdditionalPrinterColumns(v1alpha1.AdditionalPrinterColumnPolicyReplace, additionalPrinterColumns),
```

Comment:
arent the additional printer columns here already the finished ones? why do you need to reuse the method here? You already set the final columns in https://github.com/kro-run/kro/pull/653/files#diff-4be45bed5efa573c9cfcf7c8eec372cb4de316a24e6709f1bbda6e60722685a0R35 no?